### PR TITLE
adding query string values test for seq page

### PR DESCRIPTION
**Pull request by ScottDowne**

### CODE REVIEW

**Comment by ScottDowne** [comment]
@alicoding r?

File: tests/react/sequential-props.tests.jsx [hunk 1]
```diff
@@ -0,0 +1,80 @@
+var React = require('react');
+var TestUtils = require('react/lib/ReactTestUtils');
+var should = require('should');
+var Context = {
+  messages: require('../../locales/en-US.json'),
+  router: function() {}
```
Comment (alicoding) [inline tests/react/sequential-props.tests.jsx]:
```
router: function() {
  return {
    ....
  }
}
```

File: tests/react/sequential-props.tests.jsx [hunk 2]
```diff
@@ -0,0 +1,80 @@
...
+};
+Context.router.makeHref = function() {};
```
Comment (alicoding) [inline tests/react/sequential-props.tests.jsx]: Move them all up there.

File: tests/react/sequential-props.tests.jsx (hunk 1 above)
Comment (ScottDowne) [inline tests/react/sequential-props.tests.jsx]: I tried that, it's not a function that returns an object, it's a function acting as an object. It's weird, but js allows it.

Comment (alicoding) [inline tests/react/sequential-props.tests.jsx]: Then do it this way instead.
```
assign(Context.router,
  {
    makeHref: function() {},
    makePath: function() {},
    transitionTo: function() {},
    replaceWith: function() {},
    goBack: function() {},
    getCurrentPath: function() {},
    getCurrentRoutes: function() {},
    getCurrentPathname: function() {},
    getCurrentParams: function() {},
    getCurrentQuery: function() {},
    isActive: function() {},
    render: function() {}
  });
```

Comment (ScottDowne) [inline tests/react/sequential-props.tests.jsx]: Yeah, I'm fine with that.

**Pull request merged**